pkg/output/stdout.go

```diff
@@ -196,10 +205,59 @@ func (s *Stdoutput) Warning(warnstring string) {
 	}
 }
 
+func (s *Stdoutput) writeToAll(config *ffuf.Config, res []Result) error {
+	var err error
+	var BaseFilename string = s.config.OutputFile
+
+	// Go through each type of write, adding
+	// the suffix to each output file.
+
+	s.config.OutputFile = BaseFilename + ".json"
+	err = writeJSON(s.config, s.Results)
+	if err != nil {
+		s.Error(fmt.Sprintf("%s", err))
+	}
+
+	s.config.OutputFile = BaseFilename + ".ejson"
+	err = writeEJSON(s.config, s.Results)
+	if err != nil {
+		s.Error(fmt.Sprintf("%s", err))
+	}
+
+	s.config.OutputFile = BaseFilename + ".html"
+	err = writeHTML(s.config, s.Results)
+	if err != nil {
+		s.Error(fmt.Sprintf("%s", err))
+	}
+
+	s.config.OutputFile = BaseFilename + ".md"
+	err = writeMarkdown(s.config, s.Results)
+	if err != nil {
+		s.Error(fmt.Sprintf("%s", err))
+	}
+
+	s.config.OutputFile = BaseFilename + ".csv"
+	err = writeCSV(s.config, s.Results, false)
+	if err != nil {
+		s.Error(fmt.Sprintf("%s", err))
+	}
+
+	s.config.OutputFile = BaseFilename + ".csv"
+	err = writeCSV(s.config, s.Results, true)
```

Comment:
I believe this is supposed to call eCSV instead of CSV (also with a correct file extension)

Reply:
Good catch, the file extension was a mistake.  However, it is still the writeCSV function, but with true or false.  Fixing now!